### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by aaron-ai**

### CODE REVIEW

File: instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/cache/concurrentlinkedhashmap/LinkedDeque.java [hunk 1]
```diff
@@ -31,9 +31,9 @@
 
 /**
  * Linked list implementation of the {@link Deque} interface where the link pointers are tightly
- * integrated with the element. Linked deques have no capacity restrictions; they grow as necessary
- * to support usage. They are not thread-safe; in the absence of external synchronization, they do
- * not support concurrent access by multiple threads. Null elements are prohibited.
+ * integrated with the element. Linked dequeues have no capacity restrictions; they grow as
```
Comment (mateuszrzeszutek) [inline instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/cache/concurrentlinkedhashmap/LinkedDeque.java:34]: This one was actually on purpose "deques", as in Java `Deque` interface, the double ended queue

**Review by mateuszrzeszutek — COMMENTED** [review]
Thanks!
Left one minor comment, but overall it LGTM.

File: instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/cache/concurrentlinkedhashmap/LinkedDeque.java (hunk 1 above)
Comment (aaron-ai) [reply instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/cache/concurrentlinkedhashmap/LinkedDeque.java]: fixed.

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by trask — APPROVED** [review]
thx 👍

**Pull request merged**